File: backend/models/todo_model.go
package models

type TodoModel struct {
	ID          int      `json:"id"`
	Title       string   `json:"title"`
	Description string   `json:"description"`
	IsDone      bool     `json:"isDone"`
	Tags        []string `json:"tags"`
	User        string   `json:"user"`
	Time        string   `json:"time"`
}

File: frontend/lib/models/todo_model.dart
// ignore_for_file: non_constant_identifier_names

class TodoModel {
  int? id;
  String title;
  String? description;
  bool isDone;
  String? tag;
  String User;
  DateTime time;

  TodoModel(
      {this.id,
      required this.title,
      this.description,
      required this.isDone,
      this.tag,
      required this.User,
      required this.time});
}
